ebos/eclbasevanguard.hh

```diff
@@ -235,10 +245,14 @@ public:
 
         EWOMS_REGISTER_PARAM(TypeTag, bool, OwnerCellsFirst,
                              "Order cells owned by rank before ghost/overlap cells.");
+#if HAVE_MPI
         EWOMS_REGISTER_PARAM(TypeTag, bool, SerialPartitioning,
                              "Perform partitioning for parallel runs on a single process.");
         EWOMS_REGISTER_PARAM(TypeTag, double, ZoltanImbalanceTol,
                              "Tolerable imbalance of the loadbalancing provided by Zoltan (default: 1.1).");
+        EWOMS_REGISTER_PARAM(TypeTag, std::string, ZoltanParams,
+                             "Configuration of Zoltan partitioner. Valid options are: graph (default)), hypergrah or scotch. Alternatively, you can request a configuration to be read from a JSON file by giving the filename here, ending with '.json.'");
```

Comment:
Can we somehow break this line?

Nobody will know the correct option names to use for the json file without researching the Zoltan user manual. Maybe provide a link?

I also wonder what the error message (if there is one at all) will look like if there are typos.

Reply:
e.g.
```
Error: foo.json(2): expected ':'
```
and no scotch (actually parmetis in my case);
```
[0] Zoltan ERROR in Zoltan_Graph (line 103 of /build/trilinos-GM3Kol/trilinos-12.14.1/packages/zoltan/src/tpls/third_library.c):  ParMETIS partitioning was requested but Zoltan was compiled without ParMETIS.
```
followed by application abort.

Reply:
^ me with the wrong login.

Reply:
> Error: foo.json(2): expected ':'

Nothing to add... :wink: